### ISSUE TITLE
No items found. / Ingen enheder fundet

### ISSUE THREAD

**Issue by Molle-DK** [PatrickE94/com.zaptec#38]
It cant find my Zaptec Go. Maybe because i am using the Spirii Go app?

**Comment by philipostli** [comment]
Hi.

Can you please send us some info about your charger?
If you go to this site: https://api.zaptec.com/help/index.html#/Charger/get_api_chargers

- Click on Authorize, Put in username and password (should be safe to use).
- Go to api/charger (first on under "Chargers" headline)
- Click white button to the right "Try it out"
- Scroll down to blue button "Execute"
- Look at responses, next to code 200 you should see your Charger info.
- We need the ***"DeviceType"***, ***"InstallationType"*** and ***"DeviceId"*** (the other fields you should keep secret)

Then we can probably see if the app is able to recognize your charger.

**Comment by Molle-DK** [comment]
Hi. Thanks for your response

When i log in, do i need til fill the "API Key (apiKey)" ??

I don´t know if my setup is different from a normal installation. I can not charge from Zaptec Go app. I need to use the Spirii Go app, because Spirii has the control over the charger (Zaptec Go)

I can not find the precise strings you want, this is what i can find... I can copy a complete screendump if you want?



"id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "deviceId": "string"

"deviceType": 0,
  "installationName": "string",
  "installationId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",


________________________________
Fra: Philip Østli ***@***.***>
Sendt: 30. december 2024 23:19
Til: PatrickE94/com.zaptec ***@***.***>
Cc: Molle-DK ***@***.***>; Author ***@***.***>
Emne: Re: [PatrickE94/com.zaptec] No items found. / Ingen enheder fundet (Issue #38)


Hi.

Can you please send us some info about your charger?
If you go to this site: https://api.zaptec.com/help/index.html#/Charger/get_api_chargers

  *   Click on Authorize, Put in username and password (should be safe to use).
  *   Go to api/charger (first on under "Chargers" headline)
  *   Click white button to the right "Try it out"
  *   Scroll down to blue button "Execute"
  *   Look at responses, next to code 200 you should see your Charger info.
  *   We need the "DeviceType", "InstallationType" and "DeviceId" (the other fields you should keep secret)

Then we can probably see if the app is able to recognize your charger.

**Comment by philipostli** [comment]
Ok, I already see the problem. Device type 0 is not known to us. We must talk to Zaptec about this.

**Comment by Molle-DK** [comment]
Oh okay. Will you let me know what you find?

________________________________
Fra: Philip Østli ***@***.***>
Sendt: 31. december 2024 09:59
Til: PatrickE94/com.zaptec ***@***.***>
Cc: Molle-DK ***@***.***>; Author ***@***.***>
Emne: Re: [PatrickE94/com.zaptec] No items found. / Ingen enheder fundet (Issue #38)


Ok, I already see the problem. Device type 0 is not known to us. We must talk to Zaptec about this.

**Comment by philipostli** [comment]
Yes I created an [issue](https://github.com/PatrickE94/com.zaptec/issues/45) to collect all the issues related to discover correct devices. We want to fix this. Will let you know as soon as we have a fix.

**Comment by philipostli** [comment]
Fixed using deviceId. See issue #45
Wait for update
